unified-installer/astra-unified-installer.sh

```diff
@@ -2632,9 +2768,16 @@ step_add_image_remaps_to_kustomization() {
     echo "images:" >> "$kustomization_file"
     if components_include_trident || trident_operator_image_needs_upgraded; then
         # Trident operator
+        # Note: depending on which Trident YAML we pulled, the ref sometimes excludes 'docker.io', so
+        # we add remaps both with and without to be safe.
         echo "- name: netapp/$__DEFAULT_TRIDENT_OPERATOR_IMAGE_NAME" >> "$kustomization_file"
         echo "  newName: $(join_rpath "$TRIDENT_OPERATOR_IMAGE_REGISTRY" "$TRIDENT_OPERATOR_IMAGE_REPO")" >> "$kustomization_file"
         echo "  newTag: \"$TRIDENT_OPERATOR_IMAGE_TAG\"" >> "$kustomization_file"
+
+        local -r default_img="$__DEFAULT_DOCKER_HUB_IMAGE_REGISTRY/$__DEFAULT_DOCKER_HUB_IMAGE_BASE_REPO/$__DEFAULT_TRIDENT_OPERATOR_IMAGE_NAME"
+        echo "- name: $default_img" >> "$kustomization_file"
+        echo "  newName: $(join_rpath "$TRIDENT_OPERATOR_IMAGE_REGISTRY" "$TRIDENT_OPERATOR_IMAGE_REPO")" >> "$kustomization_file"
+        echo "  newTag: \"$TRIDENT_OPERATOR_IMAGE_TAG\"" >> "$kustomization_file"
```

Comment:
When testing, I found that, after a recent change to the Trident YAML we pull, the repository paths have changed from `netapp/trident-operator` to `docker.io/netapp/trident-operator`. Added the new path and leaving the old one in to be safe.